blog/2023-04-27-welcome-to-conda-dot-org.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Welcome to conda.org!"
+slug: "2022-04-27-welcome-to-conda-dot-org"
+tags: [announcement]
+description: |
+  Welcome to our new website! In this article, we talk about why we built
+  this website and how we would like to see be used going forward.
```

Comment:
```suggestion
  this website and how we would like to see it be used going forward.
```

We could alternatively put something different here, like:
"... and where we see it going as more gets added to it."

Reply:
oof, yeah I'll re-word this.